tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Models/Matter/MatterDetails.cs

```diff
@@ -43,6 +43,12 @@ public string SubareaOfLaw
             set;
         }
 
+        public IDictionary<string, ManagedColumn> ManagedColumnTerms

```

Comment:
New property ManagedColumnTerms has been added to handle dynamic taxonomy hierarchy